r/version_3/dm11_clean_participant_v3.R

```diff
@@ -280,9 +280,9 @@ dt[, part_reported_all_contacts := map_report_contacts[part_reported_all_contact
 risk_names <- grep("part_.*_risk", names(dt), value = TRUE)
 
 if(length(risk_names)>0){
-  dt[, part_med_risk_v2 := map_fm_yn[part_med_risk_v2]]	 
+  #dt[, part_med_risk_v2 := map_fm_yn[part_med_risk_v2]]	 
```

Comment:
Does this just need to be deleted then?

Reply:
Or will this affect the other data points?

Reply:
I have cross checked with the current data/clean/part.qs - does not change for the waves that are irrelevant